KerbalStuff/objects.py

```diff
@@ -165,6 +165,7 @@ class Mod(Base):
     external_link = Column(String(512))
     license = Column(String(128))
     votes = Column(Integer, default=0)
+    score = Column(Float, default=0, nullable=False)
```

Comment:
I think we can make the db side of our query faster by adding `index=True` here. This would allow the db to essentially pre-store the rows in sorted order so when we say `order_by().offset().limit()`, it could jump directly to those rows without having to access the entire table and sort.

- https://www.postgresql.org/docs/current/indexes-ordering.html